Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Yak Statue

## Changelog:
- Added Yak Statue wonder for Yaungol characters instead of Horse Statue.
- Added Yak Graveyard wonder upgrade.

## Developer changelog:
- Added yaungol_opinion opinion modifier
- Changed vanilla Great Stables wonder upgrade description into custom localization.



# How to test:
Build Yak Statue and upgrade it to the max stage. Build Yak Graveyard and Great Stables to make sure loc is working.
